Mathlib/Analysis/Convex/Topology.lean

```diff
@@ -382,8 +382,43 @@ end ContinuousSMul
 
 section LinearOrderedField
 
-variable {𝕜 : Type*} [Field 𝕜] [LinearOrder 𝕜] [IsStrictOrderedRing 𝕜]
-  [TopologicalSpace 𝕜] [OrderTopology 𝕜]
+variable {𝕜 : Type*} [LinearOrder 𝕜] [TopologicalSpace 𝕜] [OrderTopology 𝕜]
+
+open scoped Topology
+open Filter
+
+lemma nhdsWithin_diff_singleton_of_subsingleton {a : 𝕜} {s : Set 𝕜} (hs : s.Subsingleton) :
+    𝓝[s \ {a}] a = ⊥ := by
```

Comment:
This is effectively saying that `¬ AccPt a (𝓟 s)`. There's the general fact, which I *think* we don't yet have, that in a `T1Space`, `AccPt a (𝓟 s) → s.Infinite` (you build a sequence of distinct points by induction by taking a punctured neighborhood of `a` which does not contain any of the previous points in the sequence). Then your statement would be a special case of the contrapositive of this one.

Would you mind please adding these statements in another PR and then using them here?

Reply:
This should follow from the stuff in `Topology/DiscreteSubset` so I'll add it there in another PR